taas/__init__.py

```diff
@@ -141,15 +144,36 @@ def normalise_sheet(raw, mapping):
 
         Raw is the data as produced from read_sheet_csv().
         Mapping is a dict of cooked->raw pairs (eg: 'label' : 'Preferred Term')
+
+        Fields containing a . in their name will be turned into maps. Eg:
+        `label.en`, `label.es` and `label.fr` become three keys in the `label` map.
+        Only one level of embedding is supported.
```

Comment:
It's a restriction due to my n00bishness when it comes to Python. The `TODO` lines below (162-167+) are where we assume there's only one level of keys.